buildconfig/Jenkins/buildscript

```diff
@@ -340,6 +342,13 @@ if [ -e $BUILD_DIR/CMakeCache.txt ]; then
     CMAKE_GENERATOR=""
 fi
 
+###############################################################################
+# Conda R-Path setup
+###############################################################################
+if [ ${ON_CONDA} ]; then
+    CMAKE_RPATH="-DCMAKE_BUILD_RPATH=${CONDA_PREFIX}/x86_64-conda-linux-gnu/sysroot/usr/lib64"
```

Comment:
I can't quite see where `CONDA_PREFIX` gets set...

Reply:
CONDA_PREFIX is an environment variable set when we are in a conda environment, it _SHOULD_ be there if we call it in a conda build job.

Reply:
I guess I was expecting to see the conda environment setup in the script too.

Reply:
Hmmm let me think me about to Atomicity of the scripts and how I call each one in order to make it call them properly.